source/webapp/saved-recipe/saved-recipe-style.css

```diff
@@ -0,0 +1,80 @@
+@import url('https://fonts.googleapis.com/css2?family=Inknut+Antiqua&display=swap');
+
+html *{
+    font-family: Inknut Antiqua;
+}
+
+body {
+    background-color: #fffbf3;
+    font-size: 20px;
+}
+
```

Comment:
you can incorporate two css files to your page. no need to write duplicate items